### ISSUE TITLE
Column alias 'time' does not work

### ISSUE THREAD

**Issue by ideoma** [questdb/questdb#1075]
**Describe the bug**
Sometimes alias time does not work 

**To Reproduce**
```sql
create table tst (
timestamp timestamp
)

select timestamp time from tst
```

**Expected behavior**
Result column name is timestamp but should be time

**Screenshots**

**Environment (please complete the following information):**
 - Version: 6.0.2

**Additional context**
This is the workaround syntax with AS
```sql
select timestamp AS time from tst
```

**Comment by sachinmukherjee** [comment]
Can I work on this issue?

**Comment by bluestreak01** [comment]
yes, of course

**Comment by sachinmukherjee** [comment]
Hi @bluestreak01, 
Can you give me some pointers on whether this will be a fix from frontend or backend?

**Comment by ideoma** [comment]
That's backend somewhere in `ExpreessionParser` in the area of:

https://github.com/questdb/questdb/blob/61045e559e5e7146f67a8819266810963c3ecfcb/core/src/main/java/io/questdb/griffin/ExpressionParser.java#L560

**Comment by apetschenka** [comment]
As far i can see its still open? - I would work on this

**Comment by bziobrowski** [comment]
This issue seems fixed in 6.1.3-SNAPSHOT as I'm getting the following in web console :  

create table tst (
timestamp timestamp
);

select timestamp time  from tst ; 

returns 
"time"

**Comment by bsmth** [comment]
Hi @bziobrowski this was fixed in https://github.com/questdb/questdb/pull/1602 , so I will close it now. Thanks for the reminder 👍🏻
